components/member-profile/active-task/index.js

```diff
@@ -1,19 +1,62 @@
 import PropTypes from 'prop-types';
-import classNames from 'components/member-profile/active-task/active-task.module.scss';
-import { timeWas } from 'helper-functions/time-was';
+import classNames from './active-task.module.scss';
+import { timeWas } from '../../../helper-functions/time-was';
+import { percentageofDaysRemaining } from '../../../helper-functions/taskProgress';
+
 const ActiveTask = ({ taskDetails }) => {
-  const { title, purpose, startedOn, endsOn } = taskDetails;
-  let completedDate = '';
+  const { title, purpose, startedOn, endsOn, percentCompleted } = taskDetails;
+  let completedDate,
+    percentageOfDaysRemaining = '';
   completedDate = timeWas(startedOn * 1000, false, endsOn * 1000);
+
+  let percentOfTaskLeft = 100 - percentCompleted;
+
+  percentageOfDaysRemaining = percentageofDaysRemaining(startedOn, endsOn, completedDate);
+
+  function estimatedDays() {
+    if (percentageOfDaysRemaining >= percentOfTaskLeft) {
+      return classNames.showTextGreen;
+    } else if (percentageOfDaysRemaining < 50 && percentOfTaskLeft > 75) {
+      return classNames.showTextOrange;
+    } else if (percentageOfDaysRemaining < 25 && percentOfTaskLeft > 35) {
+      return classNames.showTextRed;
+    }
+    return classNames.showTextYellow;
+  }
```

Comment:
Few pointers here

- Too many if-else, we can refactor it.
- An empty line after the function.

Solution for the `if-else`

- you can create a helper function that takes the `percentageOfDaysRemaining` value as the args & returns the colors accordingly.
This approach can help us in re-using the helper function and the code readability will improve.